subsys/bluetooth/controller/include/ll_feat.h

```diff
@@ -5,6 +5,10 @@
  * SPDX-License-Identifier: Apache-2.0
  */
 
+#if defined(CONFIG_BT_CTLR_SMI_TX_SETTING)
+#include "ll_settings.h"
```

Comment:
Internal headers files do not include other internal header files. Add the includes as necessary in source files.

Reply:
Done.